File: app/src/androidTest/java/net/tokyoenvious/socialcount/source/TwitterTest.java
package net.tokyoenvious.socialcount.source;

import org.junit.Test;
import org.junit.runner.RunWith;

import static junit.framework.Assert.assertTrue;

import android.support.test.runner.AndroidJUnit4;

@RunWith(AndroidJUnit4.class)
public class TwitterTest {
    @Test
    public void fetchCountSync() throws java.io.IOException {
        Integer count = new Twitter().fetchCountSync("http://www.example.com/");
        assertTrue(count > 0);
    }
}

File: app/src/androidTest/java/net/tokyoenvious/socialcount/source/TwitterTest.kt
package net.tokyoenvious.socialcount.source

import org.junit.Test
import org.junit.runner.RunWith

import junit.framework.Assert.assertTrue

import android.support.test.runner.AndroidJUnit4

@RunWith(AndroidJUnit4::class)
class TwitterTest {
    @Test
    @Throws(java.io.IOException::class)
    fun fetchCountSync() {
        val count = Twitter("http://www.example.com/").fetchCountSync()
        assertTrue(count!! > 0)
    }
}
